### PR TITLE
ESSOptimizer: Fix recombination bug

### PR DESCRIPTION
**Pull request by dweindl**
This fixes a bug in the recombination step of the `ESSOptimizer`, also used by `SacessOptimizer`.

This will in particular improve performance when using scatter search without local optimizers.

### CODE REVIEW

File: pypesto/optimize/ess/ess.py [hunk 1]
```diff
@@ -443,11 +443,15 @@ def _combine(self, i, j) -> np.array:
             raise ValueError("i == j")
         x = self.refset.x
 
-        d = x[j] - x[i]
-        alpha = np.sign(j - i)
+        d = (x[j] - x[i]) / 2
```
Comment (PaulJonasJost) [inline pypesto/optimize/ess/ess.py:446]: why was this changed to /2?

**Review by PaulJonasJost — APPROVED** [review]
i am guessing that line 450 created the bug?

File: pypesto/optimize/ess/ess.py (hunk 1 above)
Comment (dweindl) [reply pypesto/optimize/ess/ess.py:446]: Because it was wrong :)

**Comment by dweindl** [comment]
> i am guessing that line 450 created the bug?

L446 + L450
